packages/ui/components/form/switch/Switch.tsx

```diff
@@ -19,6 +19,7 @@ const Switch = (
     fitToHeight?: boolean;
     disabled?: boolean;
     tooltip?: string;
+    small?: boolean;
```

Comment:
Having a way to shrink the switch to be shown in the lock/unlock indicator in each field